app/javascript/packs/new_ad.js

```diff
@@ -0,0 +1,137 @@
+$(document).ready(function() {
+  var maxWords = 1000;
+
+  // Initial update
+  updateRemainingWords();
+
+  // Update on input
+  $('#ad-description').on('input', updateRemainingWords);
+
+  function updateRemainingWords() {
+    var currentWords = $('#ad-description').val().match(/\S+/g) || [];
+    var remainingWords = maxWords - currentWords.length;
+
+    // Display the remaining words count
+    $('#remaining-words').text(remainingWords);
+  }
+});
+
+
+$(document).ready(function() {
+  $('#toggleButton').on('click', function() {
+    var suggestion = $('#suggestion');
+    var buttonText = $('#toggleButton');
+
+    if (suggestion.css('overflow') === 'hidden') {
+      suggestion.css({
+        'overflow': 'visible',
+        'height': 'auto'
+      });
+      buttonText.text('Show Less');
+    } else {
+      suggestion.css({
+        'overflow': 'hidden',
+        'height': '88px'
+      });
+      buttonText.text('Show More');
+    }
+  });
+});
+
+// app/assets/javascripts/posts.js
+$(document).ready(function() {
+    var textField = $('.validate-on-click');
+  var errorMessage = $('.error-message');
+
+  textField.on('focus', function() {
+    // Reset styles on focus
+    textField.css('background-color', '');
+    errorMessage.text('');
+  });
+
+  textField.on('change keydown paste input', function() {
+    var content = textField.val();
+    if (content.length >= 0 && content.length < 2 ) {
+      // Display validation error
+      errorMessage.text('Enter a valid price');
+      $($(".para5-text")[0]).removeClass("display-none")
+    } else if(content.length > 1) {
+
+      // Convert the input value using num2str plugin
+      var result = $.fn.num2str(Number(content));
+
+      // Display the converted result
+
+      $('.para5-text').text( result);
+    
+    } 
+  });
+});
+
+function convertNumber() {
```

Comment:
Where this function is called?

Reply:
i remove it

Reply:
Alright, make a another commit for that and push it